pkgs/phps.nix

```diff
@@ -33,6 +33,17 @@ let
                 url = "https://bugs.php.net/patch-display.php?bug_id=76826&patch=bug76826.poc.0.patch&revision=1538723399&download=1";
                 sha256 = "aW+MW9Kb8N/yBO7MdqZMZzgMSF7b+IMLulJKgKPWrUA=";
               })
+            ]
+            ++ prev.lib.optionals (prev.lib.versionOlder args.version "7.4" && prev.stdenv.isDarwin) [
```

Comment:
Since the patch applies on all platforms, it might be better to apply it on all platforms to prevent bit rot.

Reply:
Fixed in 8aae443b1909450a4e3f807af7bbe06aeaf3ebe9, you may squash merge in order to keep the history clean.

Reply:
Fixed in 8aae443b1909450a4e3f807af7bbe06aeaf3ebe9, you may squash merge in order to keep the history clean.